server/channels/api4/file_test.go

```diff
@@ -832,9 +838,9 @@ func TestGetFileHeaders(t *testing.T) {
 	t.Run("txt", testHeaders(data, "test.txt", "text/plain", false, false))
 	t.Run("html", testHeaders(data, "test.html", "text/plain", false, false))
 	t.Run("js", testHeaders(data, "test.js", "text/plain", false, false))
-	if os.Getenv("IS_CI") == "true" {
+	if os.Getenv("IS_CI") == "true" || runtime.GOOS == "darwin" {
```

Comment:
This has just been broken for a while, confirmed with a bunch of MacOS developers: https://community.mattermost.com/private-core/pl/jcnjfaxqbtnwmpjzzyjt8jqw4r.